linux/lvm.go

```diff
@@ -201,9 +205,8 @@ func (ls *linuxLVM) HasPV(name string) bool {
 }
 
 func (ls *linuxLVM) CreateVG(name string, pvs ...disko.PV) (disko.VG, error) {
-	const mdSize = 128 * disko.Mebibyte
 	cmd := []string{"lvm", "vgcreate",
-		fmt.Sprintf("--metadatasize=%dB", mdSize),
+		fmt.Sprintf("--metadatasize=%dB", pvMetaDataSize),
```

Comment:
Does the VG metadata size need to match the PV metadata size? or would the VG metadata size value match what the PV has set?

Reply:
> Does the VG metadata size need to match the PV metadata size? or would the VG metadata size value match what the PV has set?

the --metadatasize *is* the PV size.  it is just ignored entirely for pvs thave have already been 'pvcreat'ed.